website/docs/targeting/targeting-rule/user-condition.md

```diff
@@ -0,0 +1,196 @@
+| ---------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| BEFORE     | Checks whether the comparison attribute interpreted as a second-based [Unix timestamp](https://en.wikipedia.org/wiki/Unix_time") is less than the comparison value.    |
+| AFTER      | Checks whether the comparison attribute interpreted as a second-based [Unix timestamp](https://en.wikipedia.org/wiki/Unix_time") is greater than the comparison value. |
+
+#### Array Comparators
+
+The following comparators expect the *Comparison attribute* to be an array of strings (or an array of strings serialized to JSON), and the *Comparison value* to be a list of strings.
+
+| Comparator                | Description                                                                                                                    |
+| ------------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| ARRAY CONTAINS ANY OF     | Checks whether the comparison attribute interpreted as an array of strings contains any of the comparison values.        |
+| ARRAY NOT CONTAINS ANY OF | Checks whether the comparison attribute interpreted as an array of strings does not contain any of the comparison values |
+
+## Examples
+
+### Black Friday Sale
+
+#### Context
+Black Friday is coming and we want to offer a special discount to our users.
+
+#### Goal
+The deals should become available at midnight on Black Friday. We want to make sure that the users can't access the deals before that.
+
+#### Solution
+Let's use the AFTER comparator to check whether the current time is after midnight on Black Friday.
+
+On the Dashboard:
+![Black Friday Sale Example](/assets/targeting/targeting-rule/user-condition/black-friday-example.jpg)
+
+In the code:
+```js
+const userObject = {
+  identifier: '<SOME USER ID>',
+  email: userEmail,
+  // In case this code runs on the customer's device, we should consider getting the current time
+  // from a time server since customers are free to set the system clock on their devices.
+  custom: { Date: new Date() },
+};
+const value = await configCatClient.getValueAsync("enableBlackFridayDeals", false, userObject);
+```
+
+### Confidential CONTAINS and NOT CONTAINS workaround
+
+#### Context
+We use ConfigCat in our Angular frontend application to control the availability of a feature.
+
+#### Goal
+We want to enable this feature for users from certain companies. To do the job, we would need to use the `CONTAINS` or `NOT CONTAINS` comparators.
+
+However, there is no confidential version of these comparators and we are well aware that, for privacy reasons, it's not a good idea to use cleartext comparators for applications running in the browser.
+
+How can we solve this?
+
+#### Solution
```

Comment:
Erre talán jobb példa lenne az ends with any of-ot használni, annak pont ez volt a fő use case-e.

Reply:
Valóban, a példa nem teljesen találó, mert most már az ends with any of-fal ezt a konkrét szituációt egyszerűbben is meg lehet oldani. (Gondolom azért sikerült így, mert ez még a régi doksiból jön: https://configcat.com/docs/advanced/targeting/#confidential-text-comparators)

Mondjuk ettől függetlenül vsz. ki lehetne találni egy olyan példát, ami csak ezzel a módszerrel működik. Viszont ehhez @sigewuzhere közreműködésére is szükség lenne a screenshotok miatt, szóval szerintem ezt írjuk most fel todo-ba, aztán oldjuk majd meg egy későbbi PR-ban.